projects/kit/components/textarea/textarea.style.less

```diff
@@ -153,7 +153,7 @@
     pointer-events: none;
     color: transparent;
     overflow: hidden;
-    border: solid transparent;
+    border: 0 solid transparent;
     border-top: 0;
```

Comment:
Looks like we don't need these 2 lines now?

Reply:
yes, fixed